app/views/orga/teams/show.html.slim

```diff
@@ -82,6 +84,17 @@ nav.btn-group
       = links_to_conferences([preference.conference]).first
       = preference_option(preference)
 
+- if can? :see_offered_conferences, @team
+
+  - if @team.conference_attendances.present?
+    h4 Conference offers
+    dl
+      - @team.conference_attendances.each do |offer|
+        dt Offered conference:
+        dd = offer&.conference&.name
```

Comment:
yeah, that is true. I forgot to remove it. I just put it because before add a reject_if I had an offer without conference that was giving me some errors.